package.json

```diff
@@ -15,12 +15,17 @@
     "httpntlm": "^1.5.2",
     "request": "^2.60.0",
     "underscore": "^1.8.3",
-    "underscore-deep-extend": "0.0.5",
+    "underscore-deep-extend": "github:kurtmilam/underscoreDeepExtend",
```

Comment:
Does this work without the http://? Haven't seen this syntax before!
Saw your issue raised against the repo to publish, in the meantime maybe we could point to a tarball, to prevent breaking changes on master causing issues?
I think it'd be
`https://github.com/kurtmilam/underscoreDeepExtend/tarball/359392922c5532c4d90a30536ab0c238cbc77fd2`